### DISCUSSION TITLE
Can you tell me step by step to add Airsim to Unreal Engine 5

### DISCUSSION THREAD

**Discussion by vanducdhtm** [microsoft/AirSim#4606] [Support Q&A]
I have installed UE 5.0.2 and I build Airsim like the direction [https://microsoft.github.io/AirSim/build_windows/](url)

I create a new project ue5. After, I copy the Plugins folder (Airsim/Unreal), and all folders and files in Airsim/Unreal/Environments/Blocks to my project.

When I open a project and rebuild Airsim, I see an error
![anhnhap](https://user-images.githubusercontent.com/90029085/177496954-a11656ed-9485-4a15-85ed-d6ebea1d488d.jpg)

Can you tell me step by step to add Airsim to Unreal Engine 5?

**Comment by czero69** [comment]
with few tweaks, I was able to run Airsim computer vision mode in UE 5.0.1 and 5.0.2:
https://github.com/czero69/AirSim/tree/ue5-cv
With similar approach, I was also able to run Airsim for the most recent UE5 main branch (5.1), but it needs more changes than above branch if I remeber correctly.

Please note that this will allow you to run only computer vision mode.
